config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/SafetyFirst-AAAAAAAAAAAAAAAAAAAIcQ==.json

```diff
@@ -31,7 +31,7 @@
+      "desc:8": "Here are some tips and warnings about electricity. \n\nMachines out in the rain will start fires nearby. Fires next to a machine have a random chance of an explosion. Also, sending a higher voltage than a machine\u0027s tier causes an explosion. Or several of them if they used the same cable. \n\nSending more Amperes than a machine can use is fine. However, if a cable can\u0027t handle that amperage or voltage, it will burn up and start fires that may cause - you guessed it - machine explosions. As a rule of thumb, a cable should be able to carry as much amps as the sum of all generators and emitting buffers connected to it.\n\nA generator that supplies 1A can feed multiple machines if their recipes require less than the full voltage. For example, a Macerator needs only 2 EU/t most of the time, so you could run several of them from a single 1A 32 EU/t generator, if the wires aren\u0027t too long. The wire length matters because there is loss per-amp-per-block, listed on the wire or cable. Cables lose less than wires. Note that sometimes a recipe needs a lot more EU than you expect, like a macerator sand -\u003e quartz dust recipe.\n\nMost machines only use 1A, but there are exceptions; Arc Furnace, Thermal Centrifuge, World Accelerator, and the UUM machines. Machines can accept more amps than they use - usually 2A - in order to top up their internal buffer, especially if they didn\u0027t receive most of the amp because of loss, which is a problem mostly in LV. The formula is (EU/t of Recipe * 2) / Voltage Tier + 1.\n\nSome machines are specifically there to help you deal with multiple amps, such as battery buffers, transformers, and diodes. Check out the quest tab Storing and Transforming EU for more on those.\n\nIf you want to send more amps, use a larger cable or superconductor. Later in the progression, you can use laser hatches to send even more amps, or even a wireless option.\n\nA single block machine starved of EU will restart progress but not lose the recipe items. A multiblock machine §owill§r lose the items.\n\nAny block that emits EU has loss when doing so of 2^(tier-1) per amp. So 1 EU at LV, 2 EU at MV, and so on. Therefore, using large numbers of battery buffers in a row is very wasteful. \n\nThe losses are especially important when you make your first Electric Blast Furnace (EBF) later in LV. Maybe power it directly? Of course, if you want to keep your amps over longer distances, you\u0027ll need to place Battery Buffers (BBs) occasionally to top up the amps, although it is less of a problem the further in progression you get.\n\nPlease refer to §1§n[url]gtnh.miraheze.org/wiki/Electricity[/url]§r for more info."
```

Comment:
the old one is better to read and more accurate too. just revert.

Reply:
Could you please give examples?

The main idea was to add the info that a machine doesn't blow up from overamp. The rest of the changes can be dropped if you so desire.

Reply:
done in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/1c80708d6249d083b9f7e06e58b3180ab77d5e3d